react/react dependency installer/8f8bd450-3f2b-45a5-abcc-12b633899c77.json

```diff
@@ -0,0 +1,127 @@
+{
+  "name": "react/React Dependency Installer",
+  "description": "Installs dependencies defined by the user for the react installer node. Node uses npm to install the dependencies on the same application path as the react application.",
+  "interface": {
+    "inputs": {
+      "1fc459fc-545e-4675-9916-b94bcab3563c": {
+        "type": "string[]",
+        "name": "dependencies",
+        "description": "Array of npm dependencies to install.\n\neg.\n\n[\"a\",\"b\"]"
+      },
+      "9484b15d-ca86-4d63-b07f-cb4f8f21e88b": {
+        "type": "string",
+        "name": "projectPath",
+        "description": "Installation path for npm installer.\n"
+      },
+      "6668cb7f-874b-4c24-ae30-89fe2e24bad1": {
+        "type": "any",
+        "name": "run",
+        "description": "Run npm installer if boolean value received is true.\n"
```

Comment:
Now that the type of `run` is `any`, it runs on any received value, not just true, and the description should reflect that.